cmds/core/tr/tr.go

```diff
@@ -141,9 +162,9 @@ func runesToRunes(in []rune, out ...rune) *transformer {
 	}
 }
 
-func (t *transformer) run(r io.Reader, w io.Writer) error {
-	in := bufio.NewReader(r)
-	out := bufio.NewWriter(w)
+func (c *command) run() error {
+	in := bufio.NewReader(c.stdin)
```

Comment:
or, possibly, run() could have two parameters, io.Reader and io.Writer, not sure. wdyt?

Reply:
both kinda fine, but let's stick with newCommand(in io.Reader, out io.Writer, args []string, del bool)